src/import/YAMLConfiguration.ts

```diff
@@ -166,6 +166,9 @@ export type YAMLConfiguration = {
   // * status: ci-build
   // * current: true
   history?: YAMLConfigurationHistory;
+  // When the FSHOnly parameter is set to true, no IG specific content will be generated, SUSHI will
+  // only convert FSH definitions to JSON files. When false or unset, IG content is generated.
```

Comment:
Should this comment just say "when false" now?

Reply:
Technically no here I think, because in the YAML that the user is giving as input, the `FSHOnly` can be unset.

Reply:
Ohhh right right